backend/app/crud/followers.py

```diff
@@ -0,0 +1,108 @@
+""" Followers related CRUD methods """
+from typing import Any
+from sqlmodel import Session, select
+from app.models import Followers, FollowersCreate, FollowersUpdate, User, UserCreate, UserUpdate
```

Comment:
```suggestion
from app.models import Followers, User, UsersOut, UserOut
```
Imports innecesarios eliminados, y añadidos UsersOut y UserOut